tools/toolchains/mbed_toolchain.py

```diff
@@ -1086,6 +1086,14 @@ def dump_build_profile(self):
             self._overwrite_when_not_equal(where, json.dumps(
                 to_dump, sort_keys=True, indent=4))
 
+    def check_minimal_printf(self, target):
```

Comment:
The `check` alone made me think that a value should be returned, I suggest the following method name:

```suggestion
    def check_and_add_minimal_printf(self, target):
```

Reply:
[Done](https://github.com/ARMmbed/mbed-os/compare/074bab34e361c1eebb64aa4aecd9d92903afabb6..5dd0401a5dc0debb5c42b54a63acdb1414670372).